### DISCUSSION TITLE
Stellarium installato su Eagle 4S

### DISCUSSION THREAD

**Discussion by paolo1953** [Stellarium/stellarium#1611] [Q&A]
Buongiorno
Ho istallato Stellarium su Eagle 4S ma non funziona,  non c'è il controllo del telescopio....., dopo scelta la porta Com si attiva il collegamento ma non appare il puntatore.....   

 Cosa sbaglio...?
Grazie 
Saluti 
                                                  paolo

**Comment by gzotti** [comment]
Step 1: Make your question international. Google translate yields:

Hello I have installed Stellarium on Eagle 4S but it does not work, there is no telescope control ....., after choosing the Com port the connection is activated but the pointer does not appear ..... What am I wrong ...? 

Step 2: Explain what is an Eagle 4S? You mean this? https://www.argos.co.uk/product/7829277

Telescope control is explained in the User Guide.

**Comment by paolo1953** [comment]
No questo  https://www.primalucelab.com/astronomia/computer-e-software/eagle4-s-unita-di-controllo-avanzata-per-telescopi-e-astrofotografia.html
Saluti 
                             paolo

Inviato da Posta per Windows 10

Da: Georg Zotti
Inviato: domenica 18 aprile 2021 14:29
A: Stellarium/stellarium
Cc: paolo1953; Author
Oggetto: Re: [Stellarium/stellarium] Stellarium installato su Eagle 4S (#1611)

Step 1: Make your question international. Google translate yields:
Hello I have installed Stellarium on Eagle 4S but it does not work, there is no telescope control ....., after choosing the Com port the connection is activated but the pointer does not appear ..... What am I wrong ...?
Step 2: Explain what is an Eagle 4S? You mean this? https://www.argos.co.uk/product/7829277
Telescope control is explained in the User Guide.
